hawkular-apm-server/standalone-wrapper.sh

```diff
@@ -13,8 +13,20 @@ elif [ $RT -ne 1 ]; then
   echo "An error has been found when attempting to check if an admin user exists. Aborting."
   exit 1
 else
-  username=${HAWKULAR_APM_ADMIN_USERNAME:-"admin$(head /dev/urandom -c 512 | tr -dc A-Z-a-z-0-9 | head -c 7)"}
-  password=${HAWKULAR_APM_ADMIN_PASSWORD:-$(head /dev/urandom -c 512 | tr -dc A-Z-a-z-0-9 | head -c 17)}
+  if [ -f "/client-secrets/hawkular-apm.username" ] ; then
+    username=$(cat /client-secrets/hawkular-apm.username)
+  else
+    username=${HAWKULAR_APM_ADMIN_USERNAME:-"admin$(head /dev/urandom -c 512 | tr -dc A-Z-a-z-0-9 | head -c 7)"}
+  fi
+
+  if [ -f "/client-secrets/hawkular-apm.password" ] ; then
+    password=$(cat /client-secrets/hawkular-apm.password)
+  else
+    password=${HAWKULAR_APM_ADMIN_PASSWORD:-$(head /dev/urandom -c 512 | tr -dc A-Z-a-z-0-9 | head -c 17)}
+  fi
+
+  username=${username:-"admin$(head /dev/urandom -c 512 | tr -dc A-Z-a-z-0-9 | head -c 7)"}
```

Comment:
So if the username/pass is defined in external files then it is used if not `HAWKULAR_APM_ADMIN_XXX-random` is used.

But why random string is added twice?